python/samples/kernel-syntax-examples/chat_gpt_api_function_calling.py

```diff
@@ -42,6 +42,8 @@
 kernel.import_skill(MathSkill(), skill_name="math")
 
 # enabling or disabling function calling is done by setting the function_call parameter for the completion.
+# when the function_call parameter is set to "auto" the model will decide which function to use, if any.
```

Comment:
So this introduces a requirement that a function ought never be named 'auto' -- yeah? Maybe this should be a fully qualified `SkillName.FunctionName`?

Reply:
under the covers the name being used towards openai is already skillname-functionname, since only a single word is allowed, even when using global skill that will be used in the same way as well. Better docs for using the function_call param is then def needed, good call.